File: src/main/java/com/monis/Hello.java
package com.monis;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello monis");
        Person person = new Person("monis", 66.5f, 1.7f);
        person.hello();
        /*person.name = "monis";
        person.weight = 66.5f;
        person.height = 1.7f;*/
        System.out.println(person.bmi());

      /*  //basic data type
        int age = 19;
        char c = 'A';
        byte b = 120;
        float weight = 66.5f;
        double height = 1.7;
        boolean adult = true;
        boolean enroll = false;
        //reference data type - wrapper class
        Integer age2 = 29;
        Character c2 = 'B';
        String name = "Monis";*/

    }
}

File: src/main/kotlin/com/monis/Hello.kt
package com.monis

fun main() {
//    println("Hello kotlin")
//    Human().hello()
    val human = Human(66.5f, 1.7f)
    human.hello()
    println(human.bmi())

/*    //資料型態都是物件
    val age = 19  //不可變
    val name = "Monis"
    var weight = 66.5 //可變
    var name2: String
    name2 = "monis"*/

}

class Human(var weight: Float, var height: Float, var name: String = "") {
    init {
        println("Test $weight")
    }

//    constructor(name: String, weight: Float, height: Float) : this(weight, height)

    fun bmi(): Float {
        val BMI = weight / (height * height)
        return BMI
    }

    fun hello() {
        println("Hello kotlin")
    }
}

